static/css/custom.css

```diff
@@ -1,54 +1,82 @@
 :root {
+  --step--1: clamp(0.8rem, calc(0.7rem + 0.44vw), 0.95rem);
```

Comment:
The variables without the `--ifm` namespace are related to the responsive typography and spacing from Utopia.fyi, while those with the namespace are to override the default options.